pkg/adaptor/cloud/libvirt/provider.go

```diff
@@ -0,0 +1,95 @@
+// (C) Copyright Confidential Containers Contributors
```

Comment:
A build constraint for `libvirt` is necessary here to suppress the unit test failures, since `CGO_ENABLED=0` is set on the other cloud providers.

```
//go:build libvirt
```

Reply:
I kept the libvirt functions private, but didn't add a build constraint. Fixed it now